regression/goto-harness/do-not-use-nondet-for-selecting-pointers-to-treat-as-equal/test.desc

```diff
@@ -2,11 +2,11 @@ CORE
 test.c
 --function test --harness-type call-function --treat-pointers-equal x,y --treat-pointers-equal-maybe
 should_make_equal
-\[test.assertion.1\] line 3 assertion x: SUCCESS
-\[test.assertion.2\] line 4 assertion y: SUCCESS
-\[test.assertion.3\] line 5 assertion x == y: FAILURE
-\[test.assertion.4\] line 6 assertion x != y: FAILURE
-\[test.assertion.5\] line 7 assertion \*x == \*y: FAILURE
+\[test.assertion.1\] line 3 assertion: SUCCESS
+\[test.assertion.2\] line 4 assertion: SUCCESS
+\[test.assertion.3\] line 5 assertion: FAILURE
+\[test.assertion.4\] line 6 assertion: FAILURE
+\[test.assertion.5\] line 7 assertion: FAILURE
```

Comment:
Have we lost something useful and helpful here by not showing any detail of the assertion that failed? (I realise the empty messages may contribute to this, but with the comment of @peterschrammel above perhaps we should consider having the same behaviour in our override?)

Reply:
+1 these should give a message...